### PR TITLE
Hello Microverse project pull request

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
This is a simple HTML and CSS project. We have used an h1 heading tag and styled the color red.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```
Comment (Abenezer-Tilahun) [inline style.css]: - [ ] As stated in the project requirement your .css file should includes one class with styles for your header (make it in your favorite color), Kindly update it.
Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Kashif-Saleem-Ghuman  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 👏🏻

✔️ No Linter Error
✔️ You used correct GitHub Flow. 
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Your  README file should be descriptive all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a your repo

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (Kashif-Saleem-Ghuman) [reply style.css]: I have made the changes. Kindly review

**Review by topeogunleye — APPROVED** [review]
### Hi @Kashif-Saleem-Ghuman, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
- [ ] Next time, and for future Pull Requests, please kindly add a more descriptive PR title and description using past tense as seen in the demo below:
pull_request_desc
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**